src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx

```diff
@@ -125,6 +141,14 @@ export const TxModalWrapper = ({
     onSubmit(txParameters, !doExecute)
   }
 
+  const parametersStatus: ParametersStatus = isCreation
+    ? doExecute
+      ? 'ENABLED'
+      : 'ETH_HIDDEN' // allow editing nonce when creating
+    : doExecute
+    ? 'SAFE_DISABLED'
+    : 'DISABLED' // when not creating, nonce cannot be edited
```

Comment:
Basically this is what makes ApproveTxModal compatible with the tx creation modals.

Needs to be refactored into a function at some point, and those strings into an enum. For another time.

Reply:
I think we are missing some condition here as with one owner we hide the Tx parameters
![Screen Shot 2022-01-26 at 08 54 05](https://user-images.githubusercontent.com/32431609/151125428-2ad674aa-9012-46e9-be67-2ceb768a8b13.png)

Reply:
What policy is this Safe?

Reply:
it's one owner

Reply:
are we going to fix this one?

Reply:
Yes, definitely a bug, I'm on it.

Reply:
Fixed ✅. I've also restored the tests for this function and added a test case for this.